pkgs/development/node-packages/node-env.nix

```diff
@@ -363,7 +363,7 @@ let
 
         npm ${forceOfflineFlag} --nodedir=${nodeSources} ${npmFlags} ${stdenv.lib.optionalString production "--production"} rebuild
 
-        if [ "''${dontNpmInstall-}" != "1" ]
+        if [ "$dontNpmInstall" != "1" ]
```

Comment:
Nor this.

Reply:
This comes from a tree-wide change: https://github.com/NixOS/nixpkgs/commit/2811b032d639684d3983282e9d10d89f37d161d5

It is to be expected.

@Ericson2314, this file has been auto-generated, should we upstream that to https://github.com/svanderburg/node2nix/blob/master/nix/node-env.nix#L366 ?